foundation/foundation-mda/src/main/resources/templates/deployment/configuration-store/v2/configuration-store.values.yaml.vm

```diff
@@ -0,0 +1,30 @@
+#[[########################################
+## CONFIG | Configuration Store Configs
+## See aiSSEMBLE aissemble-configuration-store-chart: README for more details.
+## https://github.com/boozallen/aissemble/
+########################################]]#
+
+aissemble-configuration-store-chart:
+
+  deployment:
+    env:
+      - name: BASE_PROPERTY_URI
+        value: /configurations/base
+      - name: ENVIRONMENT_PROPERTY_URI
+        value: /configurations/env
+
+  ingress:
+    enabled: true
```

Comment:
S: I'd consider removing this.  It's unlikely that any project will want to expose their config store over the internet. It's meant for in-cluster use.

Reply:
to that point, you think i should remove ingress all together?

Reply:
I was thinking about that.  Maybe so.  If someone was deadset on it, they could add an ingress and wire it up themselves. But I think our general advice would be to avoid it, since it's introduces a lot of security concerns that you have to be careful about.

Reply:
I think thats valid. Ill delete it and we can always add it back in later if the need arises. I would rather add it when we need it than add it now to have it never used